<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EllipseControlWidget</class>
 <widget class="QWidget" name="EllipseControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>圆心：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>（</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="circleCenterX"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>，</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="circleCenterY"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>半径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>（</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="radiansX"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>，</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="radiansY"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>长轴：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ellplseA"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>短轴：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ellipseB"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>正方形边长：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="squareLength"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>六边形边长：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hexgonLength"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>五角星边长：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="starLength"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>EllipseControlWidget</receiver>
   <slot>attribUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>updateAttrib(QVector2D,QVector2D,QVector2D,int,int,int)</signal>
  <slot>attribUpdate()</slot>
 </slots>
</ui>
